Installation/CHANGES.md

```diff
@@ -54,6 +54,8 @@ Release date: October 2023
 -   Removed the class templates `Gray_image_mesh_domain_3`, `Implicit_mesh_domain_3`, and `Labeled_image_mesh_domain_3`
     which are deprecated since CGAL-4.13.
 
+-   Added new meshing criterion `edge_distance`, an upper bound for the distance from the edge to the 1D feature.
+
```

Comment:
We need to announce a breaking change: the concept `MeshEdgeCriteria_3` has been amended.